packages/components/src/styles/global.css

```diff
@@ -12,4 +12,20 @@
   --tw-ring-shadow: 0 0 #0000;
 }
 
+/**
+ * This injects Tailwind's base styles and any base styles registered by
+ * plugins.
+ */
 @tailwind base;
+
+/**
+ * This injects Tailwind's component classes and any component classes
+ * registered by plugins.
+ */
+@tailwind components;
+
+/**
+ * This injects Tailwind's utility classes and any utility classes registered
+ * by plugins.
+ */
+@tailwind utilities;
```

Comment:
That makes sense to me! It's essentially what we're doing already (since `global.css` is only used in the storybook file), but we can be more explicit by only `purge`-ing story files. I'm less worried about the size since according to Tailwind,
> When removing unused styles with Tailwind, it’s very hard to end up with more than 10kb of compressed CSS